docs/modules/developer-guides/pages/custom-plugins.adoc

```diff
@@ -0,0 +1,76 @@
+= Cutsom Plugins
+
+VIVIDUS functionality can be extended with custom business logic. It could be
+done either in separate modules included then as dependencies, or in Gradle
+sub-projects belonging to the root test project, or in the test project directly.
+VIVIDUS defines API-s and conventions that must be followed while implementing
+custom extensions.
+
+== Dynamic variables
+
+In order to create and register own dynamic variable the following steps should
+be done.
+
+. Create a new class that implements `org.vividus.variable.DynamicVariable`:
```

Comment:
Maybe add a link to the class on GitHub?

Reply:
btw can we have a crawler to ensure links in docs are not broken?

Reply:
it worths to explore options:
https://gitlab.com/antora/xref-validator
https://github.com/owncloud/docs/blob/master/docs/checking-broken-links.md
https://github.com/timaschew/link-checker
https://github.com/Munter/netlify-plugin-checklinks